fixed-hash/src/hash.rs

```diff
@@ -526,13 +526,6 @@ macro_rules! impl_rand_for_fixed_hash {
 				ret.randomize_using(rng);
 				ret
 			}
-
-			/// Create a new hash with cryptographically random content.
-			pub fn random() -> Self {
```

Comment:
why remove? It is a breaking change

Reply:
`rustc` warned about it being unused.

Reply:
How can cargo fix know that a `pub` item is not used?